### DISCUSSION TITLE
Insertion of Table.

### DISCUSSION THREAD

**Discussion by thyanmol** [GrapesJS/grapesjs#4210] [Ideas]
First of all thanks for GrapeJs. It is great. I have a requirement of inserting table (with separate formatting for the header row and data rows). We often need to display data in tabular form. 
Also, I wanted to convert this into a document template editor library as well and not just an email template editor. If anyone could guide me, I can start working on these features.

**Comment by ronaldohoch** [comment]
Hello!

HTML Document like a simple text website or document files like .DOC, .DOCX and other?

**Reply by thyanmol** [reply]
Hi,
I want to use document files (so final design can be exported into docx format)

**Reply by ronaldohoch** [reply]
Well, good luck! I would like to see it in action when it's done.
